README.md

```diff
@@ -32,8 +32,8 @@ Applications system for Hackathons.
 Run the following commands in the console:
 
 ```
-$ git clone https://github.com/unicsmcr/hs_application.git
-$ cd hs_application
+$ git clone https://github.com/unicsmcr/hs_hs_apply.git
```

Comment:
```suggestion
$ git clone https://github.com/unicsmcr/hs_apply.git
```

There's a typo here and in the line below

Reply:
Oops, thanks!